CONTRIBUTING.md

```diff
@@ -0,0 +1,85 @@
+
+* \#bot-testing - bot-related questions (i.e. APIs)
+* \#python - language specific questions
+* \#projects - general implementation questions
+
+## How Can I Contribute?
+
+### Reporting Bugs
+
+Bug reports are tracked as [GitHub issues](https://github.com/UQComputingSociety/uqcsbot/issues). When you are creating a bug report, please include as many details as possible about what the problem is and how to reproduce it. This helps maintainers and the community understand your report :pencil:, reproduce the behavior :computer:, and find related reports :mag_right:. If an issue already exists for the bug, add any additional helpful details as a comment on that issue. Even better, you can [submit a Pull Request](#submit-pr) with a fix!
+
+> **Note:** If you find a **Closed** issue that seems like it is the same thing that you're experiencing, open a new issue and include a link to the original issue in the body of your new one.
+
+### Suggesting Enhancements
+
+Enhancement suggestions are tracked as [GitHub issues](https://github.com/UQComputingSociety/uqcsbot/issues). When creating an enhancement issue, please provide the following information:
+
+* **Use a clear and descriptive title** for the issue to identify the suggestion.
+* **Provide a step-by-step description of the suggested enhancement** in as many details as possible.
+* **Provide specific examples to demonstrate the steps**. Include copy/pasteable snippets which you use in those examples, as [Markdown code blocks](https://help.github.com/articles/markdown-basics/#multiple-lines).
+* **Describe the current behavior** and **explain which behavior you expected to see instead** and why.
+* **Include screenshots and animated GIFs** which help you demonstrate the steps. You can use [this tool](https://www.cockos.com/licecap/) to record GIFs on macOS and Windows, and [this tool](https://github.com/colinkeenan/silentcast) or [this tool](https://github.com/GNOME/byzanz) on Linux.
+* **Explain why this enhancement would be useful** to most users.
+
+### Your First Code Contribution
+
+Unsure where to begin contributing to uqcsbot? You can start by looking through these `entry level` and `help-wanted` issues:
+
+* [Entry level issues](https://github.com/UQComputingSociety/uqcsbot/issues?q=is%3Aissue+is%3Aopen+label%3A%22entry+level%22) - issues which should only require a few lines of code, and a test or two.
+* [Help wanted issues](https://github.com/UQComputingSociety/uqcsbot/issues?q=is%3Aissue+is%3Aopen+label%3A%22help+wanted%22) - issues which should be a bit more involved than `entry level` issues.
+
+#### Local development
+
+See the [README](README.md) for getting started on local development.
+
+### <a name="submit-pr"></a> Pull Requests
+
+1. Fork the UQComputingSociety/uqcsbot repo.
+1. Make your changes in a new git branch:
```

Comment:
I'd probably add in a step about calling `git clone` here

Reply:
Done :^)